nextcord/abc.py

```diff
@@ -1318,6 +1326,8 @@ async def send(
         reference: Optional[Union[Message, MessageReference, PartialMessage]] = None,
         mention_author: Optional[bool] = None,
         view: Optional[View] = None,
+        flags: Optional[MessageFlags] = None,
+        suppress_embeds: Optional[bool] = None,
```

Comment:
Why `Optional[bool]`? Why not just make the default `False`?

Reply:
Same reason as `mention_author`, default to the API default.